src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -232,11 +234,20 @@ library BeaconChainProofs {
             "BeaconChainProofs.verifyWithdrawal: historicalSummaryIndex is too large"
         );
 
-        require(
-            withdrawalProof.withdrawalProof.length ==
-                32 * (EXECUTION_PAYLOAD_HEADER_FIELD_TREE_HEIGHT + WITHDRAWALS_TREE_HEIGHT + 1),
-            "BeaconChainProofs.verifyWithdrawal: withdrawalProof has incorrect length"
-        );
+        if(block.timestamp >= denebForkTimestamp){
+            require(
+                withdrawalProof.withdrawalProof.length ==
+                    32 * (EXECUTION_PAYLOAD_HEADER_FIELD_TREE_HEIGHT_CAPELLA + WITHDRAWALS_TREE_HEIGHT + 1),
+                "BeaconChainProofs.verifyWithdrawal: withdrawalProof has incorrect length"
+            );
```

Comment:
Surely this is the wrong constant to use here? It should be the deneb constant right?